test/common/config/dummy_config.proto

```diff
@@ -0,0 +1,31 @@
+// Provides protos for testing source/common/config/config_provider_impl.{h,cc}.
+
+syntax = "proto3";
+
+package test.common.config;
+
+import "google/protobuf/timestamp.proto";
+
+message DummyConfig {
+  string a = 1;
+}
+
+message DummyConfigsDump {
+  message StaticConfigs {
+    DummyConfig dummy_config = 1;
+
+    google.protobuf.Timestamp last_updated = 2;
+  }
+
+  message DynamicConfigs {
+    string version_info = 1;
+
+    DummyConfig dummy_config = 2;
+
```

Comment:
Nit: prefer dense spacing for these protos that don't have comments or logical distinction.

Reply:
Done.